### PR TITLE
ctr: add page

### PR DESCRIPTION
**Pull request by hellojukay**
- [X] The page (if new), does not already exist in the repo.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [ ] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Review by owenvoke — APPROVED** [review]
LGTM. 👍🏻

File: pages/linux/ctr.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# ctr
+
+> Manage Containerd containers and images.
+> More information: <https://containerd.io/>.
+
+- List all containers (running and stopped):
+
+`ctr c list`
```
Comment (navarroaxel) [inline pages/linux/ctr.md:8]: Please use long arguments format:
```suggestion
`ctr containers list`
```

File: pages/linux/ctr.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- List all images:
+
+`ctr i list`
```
Comment (navarroaxel) [inline pages/linux/ctr.md:12]:
```suggestion
`ctr images list`
```

File: pages/linux/ctr.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+- Pull image:
+
+`ctr i pull {{image}}`
```
Comment (navarroaxel) [inline pages/linux/ctr.md:16]:
```suggestion
`ctr images pull {{image}}`
```

File: pages/linux/ctr.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+- Tag a image:
+
+`ctr i tag {{image1} {{image2}}`
```
Comment (navarroaxel) [inline pages/linux/ctr.md:20]: I tried this command and I think this is better:
```suggestion
`ctr images tag {{source_image}}:{{source_tag}} {{target_image}}:{{target_tag}}`
```

File: pages/linux/ctr.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+- List all images:
+
+`ctr i list`
+
+- Pull image:
+
+`ctr i pull {{image}}`
+
+- Tag a image:
```
Comment (navarroaxel) [inline pages/linux/ctr.md:18]:
```suggestion
- Tag an image:
```

File: pages/linux/ctr.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+- List all containers (running and stopped):
+
+`ctr c list`
+
+- List all images:
+
+`ctr i list`
+
+- Pull image:
```
Comment (navarroaxel) [inline pages/linux/ctr.md:14]:
```suggestion
- Pull an image:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Thanks @hellojukay, I left some comments.

File: pages/linux/ctr.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+
+`ctr containers list`
+
+- List all images:
+
+`ctr images list`
+
+- Pull an image:
+
+`ctr image pull {{image}}`
```
Comment (navarroaxel) [inline pages/linux/ctr.md:16]:
```suggestion
`ctr images pull {{image}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**